apps/docs/content/components/CTAForm.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: CTAForm
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/CTAForm/CTAForm.tsx
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=1377%3A30754'
+storybook: '/brand/storybook/?path=/story/components-CTAForm--playground'
+description: Use CTAForm to create a form that is used to collect minimal information from users.
+---
+
+import {
+  CTAForm,
+  InlineLink,
+  FormControl,
+  TextInput,
+  Checkbox,
+} from '@primer/react-brand'
+
+```js
+import {CTAForm} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>Your work email address</FormControl.Label>
+      <TextInput placeholder="name" />
+    </FormControl>
+  </CTAForm.Input>
+  <CTAForm.Confirm>
```

Comment:
Just an FYI, that the button and text input aren't quite aligned.. probably something we need to address with the tokens so if it turns into a big deal we can always create a separate issue for it 

![image](https://user-images.githubusercontent.com/13340707/218701382-5cdc5c32-98ce-4f44-aeab-c5b20d475bb5.png)

Reply:
It looks like it is off because the button includes the border in its size and the input doesn't so while the height is the  "same" in the inspection panel the actual rendered hight is different. How do we determine which is the one that needs to be updated?

Reply:
I believe the text input is the one that should change. That said, I also think this should be a separate issue, which we can discuss with Dani and Josep too.

Reply:
Ok I will leave it out of this PR for now but it will be good to note.